src/client/datascience/raw-kernel/RawKernelConnection.ts

```diff
@@ -0,0 +1,162 @@
+import { wireProtocol } from '@nteract/messaging';
+//import { KernelMessage } from '@jupyterlab/services';
+import * as zmq from 'zeromq';
+import { IDisposable } from '../../common/types';
+
+export interface IRawKernelConnectionInfo {
+    version: number;
+    iopub_port: number;
+    shell_port: number;
+    stdin_port: number;
+    control_port: number;
+    signature_scheme: string;
+    hb_port: number;
+    ip: string;
+    key: string;
+    transport: string;
+}
+
+interface IRawJupyterMessageHeader {
+    msg_id: string;
+    username: string;
+    date: string;
+    msg_type: string;
+    version: string;
+    session: string;
+}
+
+interface IRawJupyterMessage {
+    header: IRawJupyterMessageHeader | {};
+    parent_header: IRawJupyterMessageHeader | {};
+    metadata: object;
+    content: object;
+    // tslint:disable-next-line:array-type prefer-array-literal
+    buffers: Array<Buffer>;
```

Comment:
```suggestion
    buffers: Buffer[];
```

Then we can get rid of the tslint issue